.github/workflows/test.yml

```diff
@@ -9,6 +9,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: ruby/setup-ruby@v1
+      - name: Setup Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.9"
+      - name: Setup Poetry
+        uses: abatilo/actions-poetry@v2.0.0
+        with:
+          poetry-version: "1.1.13"
```

Comment:
🤔 I wonder if we should run the tests via the Docker Image which is used for running the action.

Reply:
I'm not sure what action you're referring to?

Reply:
There's a docker file in the root of the repo: https://github.com/fac/serverless-tools/blob/main/Dockerfile - Github docs: https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action - he python, poetry and pip dependencies will need to be added there so we can.

My thinking was that instead of setting up python etc. in this workflow. It should be installed in the docker image. Then that docker image should be environment for the tests. - Perhaps not though.